keyboards/dztech/dz68rgb/dz68rgb.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#define XXX KC_NO
+#include "quantum.h"
+#define LAYOUT( \
```

Comment:
You can actually use `LAYOUT_65_ansi` here so people can make use of the community layouts in https://github.com/qmk/qmk_firmware/tree/master/layouts/community

```suggestion
#define LAYOUT_65_ansi( \
```